src/test_workflow/README.md

```diff
@@ -290,6 +315,14 @@ After the performance test completes, it will report back the test results as we
 
 The development is tracked by meta issue [#126](https://github.com/opensearch-project/opensearch-build/issues/126)
 
+#### smokeTest job
+
+It is a Jenkins job designed to run smoke tests on bundle artifacts. It retrieves the bundle artifacts using the provided manifest URL parameter and spins up a cluster with all plugins installed. The cluster is deployed using the default configuration without any customizations. Smoke tests for all components are executed sequentially, and the results are returned at the end.  
```

Comment:
```suggestion
It is a Jenkins job designed to run smoke tests on distribution artifacts. It retrieves the distribution artifacts using the provided manifest URL parameter and spins up a cluster with all plugins installed. The cluster is deployed using the default configuration without any customization. Smoke tests for all components are executed sequentially, and the results are returned at the end.  
```

Reply:
Is the cluster cleaned up after every component?

Reply:
Updated based on suggestions. Thanks.


> Is the cluster cleaned up after every component?

No. We will only clean up after all components test completed.

Reply:
That might cause conflicts right?

Reply:
We have test requirements that if it causes conflicts, we shouldn't add it into the framework.

Reply:
I dont think that is feasible in the long term. Adding @prudhvigodithi @rishabh6788 @peterzhuamazon to get some more inputs.

Reply:
Our current design involves spinning up a single cluster and accommodating non-conflicting smoke test cases for multiple plugins. Perhaps we could create a separate issue to discuss this further?

Reply:
Cut an enhancement issue here. We may revisit it to analyze its benefits further.
https://github.com/opensearch-project/opensearch-build/issues/5274